src/plugins/intel_gna/src/gna_graph_compiler.cpp

```diff
@@ -805,14 +843,17 @@ void GNAGraphCompiler::finalizeConvolution2DPrimitive(InferenceEngine::CNNLayerP
     const auto single_kernel_size = in_channels * kernelHW * convolution_precision;
 
     const auto effective_kernel_h_w = convolution._kernel_y * effective_kernel_width;
-    const auto effective_single_kernel_size = in_channels * effective_kernel_h_w * convolution_precision;
+    const auto effective_single_kernel_size =
+        in_channels / convolution._group * effective_kernel_h_w * convolution_precision;
```

Comment:
convolution._group is not checked against 0 value.

Reply:
It is, during graph validation.